### PR TITLE
[PLATFORM-1291] Add copy feature for contract address

### PR DESCRIPTION
**Pull request by tumppi**
Added contract address copy feature to data union header (on address hover) and into tile meatball menu.

In depth description can be found here: https://streamr.atlassian.net/browse/PLATFORM-1291

### CODE REVIEW

**Comment by tumppi** [comment]
@mattatgit: Tile meatball menu looks now like this for data union products
Screenshot 2020-03-13 at 15 05 59

I think it's a bit confusing to have both `Copy URL` and `Copy address` items on the menu. Should we rename `Copy address` this new one to something else? `Copy contract address` might be too long and make menu look weird. How about ordering? Should the `View *` items be grouped together and have the copy options at the bottom?

**Comment by mattatgit** [comment]
@tumppi 

Yeah good points. Maybe let's use something like this:

Edit
Copy ETH address
Copy URL
View stats
View members

File: app/src/userpages/components/Avatar/NameAndUsername/index.jsx [hunk 1]
```diff
@@ -23,9 +24,11 @@ const NameAndUsername = ({ name, username, children }: NameAndUsernameProps) =>
             <div className={styles.usernameWrapper}>
                 <span title={username} className={styles.username}>
                     {!isEthAddress && (username)}
-                    {!!isEthAddress && (truncate(username, {
-                        maxLength: 20,
-                    }))}
+                    <HoverCopy value={username}>
```
Comment (mondoreale) [inline app/src/userpages/components/Avatar/NameAndUsername/index.jsx]: This always generates a `<div>` which makes the structure dirty if we're rocking username only. I'd consider doing
```jsx
{isEthAddress ? (
    <HoverCopy …>
        {/* Here we can go with unconditioned `truncate` */}
        {truncate(username, …)}
    </HoverCopy>
) : (
    username
)}
```

^^ makes it _either-or_.

**Comment by tumppi** [comment]
Fixed things you mentioned 👍

**Pull request merged**